pandas/tests/io/parser/test_encoding.py

```diff
@@ -239,9 +239,14 @@ def test_parse_encoded_special_characters(encoding):
 def test_encoding_memory_map(all_parsers, encoding):
     # GH40986
     parser = all_parsers
+
+    # add one entry with a sepcial character
+    encoding_ = encoding or "utf-8"
+    leonardo = "Léonardo".encode(encoding_, errors="ignore").decode(encoding_)
```

Comment:
This more strict test version of the test would have failed on master with the python engine.